webapp/src/components/AuctionPage/AuctionPage.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Message,
+  Header,
+  Grid,
+  Container
+  // Button
+} from 'semantic-ui-react'
+import { utils } from 'decentraland-commons'
+
+import { parcelType } from 'components/types'
+import ParcelPreview from 'components/ParcelPreview'
+import ParcelAttributes from 'components/ParcelAttributes'
+import { t } from '@dapps/modules/translation/utils'
```

Comment:
What do you think if we start ordering the imports as we set in the upcoming standard?

Reply:
I'd like to! but when I made this the standard didn't exist. Do you mind if I start doing that on the newer PRs? to minimize conflicts